src/pages/reset-password/index.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Formik, Form } from 'formik'
+import Layout from 'components/layout/layout'
+import { localize, WithIntl } from 'components/localization'
+import { Container, SEO } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import { Input, Button } from 'components/form'
+import validation from 'common/validation'
+import { trimSpaces } from 'common/utility'
+import { BinarySocketBase } from 'common/websocket/socket_base'
+import Login from 'common/login'
+
+type EmailType = { email: string }
+
+type ErrorType = { email?: EmailType }
+
+const StyledContainer = styled(Container)`
+    text-align: center;
+    height: 100vh;
+    padding: auto 0;
+    justify-content: start;
+`
+
+const ButtonContainer = styled.div`
+    margin-top: 2rem;
+`
+
+const InputGroup = styled.div`
+    width: 40rem;
+    margin: 0 auto 3.4rem;
+`
+
+const StyledButton = styled(Button)`
+    margin: 0.8rem 0.4rem;
+`
+
+const resetValidation = (values: { email: EmailType }) => {
+    const errors: ErrorType = {}
+    const email = trimSpaces(values.email)
+    const email_error = validation.required(email) || validation.email(email)
+
+    if (email_error) {
+        errors.email = email_error
+    }
+
+    return errors
+}
+
+const resetSubmission = (values, actions) => {
```

Comment:
```suggestion
const resetSubmission = (values, actions) => {
```
Don't we need to type this?